src/settings-ui/PowerToys.Settings/Program.cs

```diff
@@ -51,6 +52,12 @@ public static void Main(string[] args)
                     IsElevated = args[(int)Arguments.ElevatedStatus] == "true";
                     IsUserAnAdmin = args[(int)Arguments.IsUserAdmin] == "true";
 
+                    if (args.Length > ArgumentsQty)
```

Comment:
I would prefer if we define a `RequiredArgumentsQty` and a `RequiredAndOptionalArgumentsQty` and check if `args.Length` is equal to `RequiredAndOptionalArgumentsQty`